### PR TITLE
Initialize components

### PR DESCRIPTION
**Pull request by OnlyUnknown**
- React has been set up
- File components has been added
- Book, Books state, Nav, and add book has been added
- Nave has link to pages

### CODE REVIEW

File: book-store/README.md
Comment (christianbiring1) [inline book-store/README.md:1]: [OPTIONAL]
- [ ] To keep your repos clean you may consider deleting some unnecessary files like this boilerplate README generated whit `CRA`

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+function App() {
+
+  return (
+    <>
+    <h1>book store</h1>
+    <Nav/>
+    <Routes>
+     <Route path='/' element={<Bookstate/>}/>
+     <Route path='/catigories'/>
```
Comment (christianbiring1) [inline src/App.jsx]: - [ ] Kindly make sure the categories component is rendered here on the URL path `categories` as you may notice nothing is being rendered.
```suggestion
     <Route path='/catigories' element={<Categories/>}/> 
```

File: src/components/bookstate.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <h2>This is Home page</h2>
+    <Book Name="Eat that frog" Author="Brian Tracy"
+    Genre="Self Improvment"
+    />
+    <Book Name="Automic habits" Author="James Clear"
+    Genre="Self Improvment"
+    />
+    <Book Name="Literary Translation" Author="Hasan ghazala"
+    Genre="Education"
+    />
```
Comment (christianbiring1) [inline src/components/bookstate.jsx]: - [ ] As specified in the project requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md#:~:text=Contains%20a%20list%20to%20renders%20individual%20books), kindly make sure your `Book` component is rendered as a child of `BookList`
like so:
```jsx
const BookList () => {
return(
arr_of_books.map(item => {
<Book book=item />
})
)
}
```
Then the booklist will be the one rendered in `BookState` component.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @OnlyUnknown , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — APPROVED** [review]
Hi @OnlyUnknown 👋👋👋, 

VeryNiceNiceGIF


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] Linters are passing👍.
- [x] `Categories` components are appropriately rendered👍.
- [x] `Book` component is rendered as a child of `booklist`👍

## Optional suggestions

[OPTIONAL] Kindly ensure your commit messages are written in imperative tense and capitalised in future changes as this is a display of professionalism👍.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**